[ADMIN] Add Release planning estimated timeline (#348)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: RELEASE-PLANNING.md

## Release Planning 
This estimated time line was approved by the Members working group on the Feb 29 meeting.

<table border="1">
    <caption>Estimated Timeline v1.0</caption>
    <thead>
        <tr>
            <th>Date</th>
            <th>Tasks</th>
            <th>Comments</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>Feb 28 - Mar 11</td>
            <td>Consistency Review</td>

>             <td>Opportunity for everyone to review what has been agreed upon so far in the working-draft branch. </br> Lock scope except <a href="https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/labels" target="_blank">(P0)</a>.</td>

Reviewer: ```suggestion
            <td>Opportunity for everyone to review what has been agreed upon so far in the working-draft branch. </br> Lock scope for v1.0.</td>
```

        </tr>
        <tr>
            <td>Mar 12 - Apr 22</td>

>             <td>Ongoing W.I.P</td>

Reviewer: ```suggestion
            <td>Drive v1.0 milestone issues to completion</td>
```

>             <td>New issues & resolution <a href="https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/labels" target="_blank">(P1)</a>: → CREATION → TRIAGE → WIP → R&A</td>

Reviewer: ```suggestion
            <td>Incoming issues will be prioritized by maintainers. Only <a href="https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/labels" target="_blank">P0 and some P1</a> issues will get added to the v1.0 milestone. Prioritization will take voting and comments in issues into consideration</td>
```

        </tr>
        <tr>
            <td>Apr 23 - Apr 30</td>
            <td>Final Consistency Review</td>

>             <td>Freeze working-draft. </br> Final review, only expected editorial comments.</td>

Reviewer: ```suggestion
            <td>Freeze working-draft. </br> Final review, only expecting editorial comments.</td>
```

        </tr>
        <tr>
            <td>May 2</td>
            <td>Release Candidate Approve by the WG Members</td>
            <td></td>
        </tr>
        <tr>
            <td>May 2 - May 31</td>
            <td>IPR Review</td>
            <td>Members may exclude any Essential Claims from their licensing commitments during this period.</td>
        </tr>
        <tr>
            <td>Jun 3</td>
            <td>Steering Committee Ratifies the Release</td>
            <td></td>
        </tr>
        <tr>
            <td>Jun 20</td>

>             <td>Onstage FOCUS Adoption stories</td>

Reviewer: ```suggestion
            <td>FinOps X: Onstage FOCUS Adoption stories</td>
```
